modules/apps/commerce/commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/portlet/shared/search/CPPriceRangeFacetsPortletSharedSearchContributor.java

```diff
@@ -111,6 +116,18 @@ private Facet _getFacet(
 			cpPriceRangeFacetsPortletInstanceConfiguration.
 				rangesJSONArrayString();
 
+		Optional<PortletPreferences> portletPreferencesOptional =
+			portletSharedSearchSettings.getPortletPreferencesOptional();
+
+		if (portletPreferencesOptional.isPresent()) {
+			PortletPreferences portletPreferences =
+				portletPreferencesOptional.get();
+
+			rangesJSONArrayString = GetterUtil.getString(
+				portletPreferences.getValue("rangesJSONArrayString", null),
+				rangesJSONArrayString);
```

Comment:
```suggestion
			rangesJSONArrayString = portletPreferences.getValue("rangesJSONArrayString", rangesJSONArrayString);
```

This should do the same, can you please try it and verify?

Thank you!